packages/button/src/css/index.ts

```diff
@@ -246,10 +252,22 @@ export default {
   [`.psds-button--iconAlign-${vars.iconAligns.right}`]: {
     flexDirection: 'row-reverse'
   },
-  [`.psds-button--iconAlign-${vars.iconAligns.right}.psds-button--not-iconOnly`]: {
+  [`.psds-button--iconAlign-${vars.iconAligns.right}.psds-button--not-iconOnly.psds-button--size-${vars.sizes.xSmall}`]: {
+    paddingRight: layout.spacingXXSmall
+  },
+  [`.psds-button--iconAlign-${vars.iconAligns.left}.psds-button--not-iconOnly.psds-button--size-${vars.sizes.xSmall}`]: {
+    paddingLeft: layout.spacingXXSmall
+  },
+  [`.psds-button--iconAlign-${vars.iconAligns.right}.psds-button--not-iconOnly.psds-button--size-${vars.sizes.small}`]: {
+    paddingRight: layout.spacingXXSmall
+  },
+  [`.psds-button--iconAlign-${vars.iconAligns.left}.psds-button--not-iconOnly.psds-button--size-${vars.sizes.small}`]: {
+    paddingLeft: layout.spacingXXSmall
+  },
+  [`.psds-button--iconAlign-${vars.iconAligns.right}.psds-button--not-iconOnly.psds-button--size-${vars.sizes.medium}`]: {
     paddingRight: layout.spacingXSmall
   },
-  [`.psds-button--iconAlign-${vars.iconAligns.left}.psds-button--not-iconOnly`]: {
+  [`.psds-button--iconAlign-${vars.iconAligns.left}.psds-button--not-iconOnly.psds-button--size-${vars.sizes.medium}`]: {
```

Comment:
Generally, if we change the selector in the css/index.ts, we'll have to change its usage in the react/index.ts.  

I don't see that accompanying change here, but I think the case is shown in this story:

http://localhost:6006/?path=/story/button-icon--medium

And it looks fine to my eye and has padding-left of 8px.  

Will you confirm that this ^ sounds reasonable to you and that you're accomplishing what you intend?

Reply:
@jaketrent I will need some help because I'm not sure what I need to do here. Before, the spacing between the icon and the edge of the button was the same for medium, small, xsmall buttons. I adjusted it so the space between the icon and edge of the button is xxSmall (4px) for xSmall and small button sizes.

Reply:
Great, happy to help.